echopype/tests/convert/test_convert_azfp.py

```diff
@@ -151,10 +152,10 @@ def test_convert_azfp_01a_different_ranges(azfp_path):
     echodata = open_raw(
         raw_file=azfp_01a_path, sonar_model='AZFP', xml_path=azfp_xml_path
     )
-    assert echodata.beam.backscatter_r.isel(frequency=0).dropna(
+    assert echodata.beam.backscatter_r.isel(channel=0).dropna(
         'range_sample'
     ).shape == (360, 438, 1)
-    assert echodata.beam.backscatter_r.isel(frequency=3).dropna(
+    assert echodata.beam.backscatter_r.isel(channel=3).dropna(
         'range_sample'
     ).shape == (360, 135, 1)
```

Comment:
Similar to the above it seems a good idea to change this to use `.sel(channel="SOME_CHANNEL")` here, for the sake of readability.